index.html

```diff
@@ -13,7 +15,218 @@
+    </nav>
+    <a id="menu-btn" aria-label="Menu"><i class="fa-solid fa-bars"></i></a>
+    <div class="only-mobile"></div>
+  </header>
+
+  <main>
+    <section class="wrapper headline headline-home">
+      <div class="title">
+        <span>"hello Random World!"</span>
+        <h1>A Completely unrelated random content</h1>
+      </div>
+
+      <div id="description" class="box">
+        <p class="box-content">
+          this page content where complete randomly generated by page on the internet.
+        </p>
+      </div>
+
+      <div id="date">
+        <h2>2022-07-25 (mon) ~ 29 (fri)</h2>
+        <span>@ some were in the internet</span>
+      </div>
+    </section>
+
+    <section class="wrapper categories">
+      <div class="title">
+        <h2>Random phrases</h2>
+        <div class="divider"></div>
+      </div>
+      <div class="content">
+        <div id="category-content-item-1" class="content-item content-item-with-hover">
+          <div class="img">
+            <img src="assets/Icons/exhibition.png" alt="">
+          </div>
+          <h3>Between a Rock and a Hard Place</h3>
+          <p class="summary">
+            <span>Meaning:</span> Being faced with two difficult choices.
+          </p>
+        </div>
+        <div id="category-content-item-2" class="content-item content-item-with-hover">
+          <div class="img">
+            <img src="assets/Icons/workshop.png" alt="">
+          </div>
+          <h3>Roll With the Punches</h3>
+          <p class="summary">
+            <span>Meaning:</span> To tolerate or endure through the unexpected mishappenings you may encounter from time
+            to time.
+          </p>
+        </div>
+        <div id="category-content-item-3" class="content-item content-item-with-hover">
+          <div class="img">
+            <img src="assets/Icons/lecture.png" alt="">
+          </div>
+          <h3>Down To The Wire</h3>
+          <p class="summary">
+            <span>Meaning:</span> A tense situation where the outcome is decided only in the last few seconds.
+          </p>
+        </div>
+        <div id="category-content-item-4" class="content-item content-item-with-hover">
+          <div class="img">
+            <img src="assets/Icons/ignite.png" alt="">
+          </div>
+          <h3>A Lemon</h3>
+          <p class="summary">
+            <span>Meaning:</span> Usually a vehicle (but any device) that you buy that's defective.
+          </p>
+        </div>
+        <div id="category-content-item-5" class="content-item content-item-with-hover">
+          <div class="img">
+            <img src="assets/Icons/forum.png" alt="">
+          </div>
+          <h3>Love Birds</h3>
+          <p class="summary">
+            <span>Meaning:</span> A pair of people who have a shared love for each other.
+          </p>
+        </div>
+      </div>
+
+      <div class="categories-btn">
+        <button type="button">See More</button>
+      </div>
+
+    </section>
```

Comment:
- [ ] Kindly change the style for this button, to match the original design.

Your design
![image](https://user-images.githubusercontent.com/35535130/181385120-d50174e5-274f-40dd-8bf3-86b3d2f8038b.png)

Original design 
![image](https://user-images.githubusercontent.com/35535130/181385169-87f968a9-4545-4a49-a8d2-ae4f9c5d6a97.png)